cmd/acadock-monitoring/main.go

```diff
@@ -59,7 +63,7 @@ func main() {
 		filters.WithAverageConfig(config.QueueLengthPointsPerSample, config.QueueLengthSamplingInterval),
 	)
 	if err != nil {
-		panic(err)
+		log.Fatalln(err)
```

Comment:
question: Why use `log.Fatalln` here rather than `panic`?

Reply:
No reason to leave it as `panic`, I was just asking by curiosity :slightly_smiling_face: